_data/simple-icons.json

```
@@ -8279,6 +8279,12 @@
             "source": "https://brand.opera.com/1472-2/opera-logos/",
             "guidelines": "https://brand.opera.com"
         },
+        {
+            "title": "Opera GX",
+            "hex": "FF1744",
```

```suggestion
            "hex": "EE2950",
```

The Opera GX icon comes out at this colour using the EPS files available in the ZIP.